src/ai/defaultai.cc

```diff
@@ -2043,11 +2044,11 @@ void DefaultAI::update_buildable_field(BuildableField& field) {
 		}
 
 		score_parts[24] =
-		   (field.unowned_land_nearby) ?
+		   (field.unowned_land_nearby) != 0u ?
```

Comment:
the brackets here seemed to be superfluous in the original as well. or they should include the comparison